CMakeLists.txt

```diff
@@ -251,84 +193,102 @@ if (NOT EXISTS ${PROJECT_SOURCE_DIR}/lib/zlib/CMakeLists.txt)
 	message(FATAL_ERROR "zlib is missing in folder lib/zlib. Have you initialized and updated the submodules / downloaded the extra libraries?")
 endif()
 
+# Include all the libraries
+# We use EXCLUDE_FROM_ALL so that only the explicit dependencies are compiled
+# (mbedTLS also has test and example programs in their CMakeLists.txt, we don't want those):
+add_subdirectory(lib/expat)
+add_subdirectory(lib/fmt)
+add_subdirectory(lib/jsoncpp EXCLUDE_FROM_ALL)
+add_subdirectory(lib/libevent EXCLUDE_FROM_ALL)
+add_subdirectory(lib/sqlite)
+add_subdirectory(lib/lua)
+add_subdirectory(lib/luaexpat)
+add_subdirectory(lib/mbedtls)
+add_subdirectory(lib/SQLiteCpp)
```

Comment:
@tigerw `SQLiteCpp` needs to be included before `lsqlite` so the `sqlite` target is available to link.